### DISCUSSION TITLE
How to set a shortcut to CocCommand document.renameCurrentWord?

### DISCUSSION THREAD

**Discussion by harvey-wei** [neoclide/coc.nvim#3680] [Q&A] [answered]

**Comment by xiyaowong** [comment]
`<plug>(coc-rename)`

**Comment by chemzqm** [comment, marked as answer]
Something like ` nmap <silent> <C-a> :call CocAction('runCommand', 'document.renameCurrentWord')<CR>`

**Reply by harvey-wei** [reply]
Thanks! It works the right way!

**Reply by harvey-wei** [reply]
> Something like ` nmap <silent> <C-a> :call CocAction('runCommand', 'document.renameCurrentWord')<CR>`

Hi! Sorry to bother you again. Is there a command like document.searchCurrentWord as a substitute for CocSearch currentword?

**Reply by chemzqm** [reply]
Write vim script like:
```viml
nnoremap <silent> <Leader>cf :exe 'CocSearch '.expand('<cword>')<CR>
```
